Master-Game/Master-Game.csproj

```diff
@@ -57,6 +58,9 @@
     <Compile Include="World\WaterTile.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="Assets\Maps\Jay01.json">

```

Comment:
I am not sure what this change means.

Reply:
I believe this is just the file you added to the project